include/tsl/sparse_hash.h

```diff
@@ -1082,9 +1091,9 @@ class sparse_hash : private Allocator,
   using sparse_buckets_allocator = typename std::allocator_traits<
       allocator_type>::template rebind_alloc<sparse_array>;
   using sparse_buckets_container =
-      std::vector<sparse_array, sparse_buckets_allocator>;
+      std::vector<sparse_array, sparse_buckets_allocator>;//here
```

Comment:
left over comment?

Reply:
You are completely correct. I removed it.